### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by david63011**
Learning objectives
Use webpack to bundle JavaScript.
Learn how to use proper ES6 syntax.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  const todoValue = input.value;
+
+  todos.push({
+    value: todoValue,
+    completed: false,
+  });
+  input.value = '';
+  renderTodos();
+}
+
```
Comment (HAMMAS-SALEEM) [inline src/index.js:32]: - [ ] You add and remove are working but i am not able to edit an existing in any way. Kindly fix it :blush:

Comment (HAMMAS-SALEEM) [inline src/index.js:32]: image

- [ ] When you will add a new todo then a todo is an object with three key-value pairs which are **description**, **completed** and **id**. But when i added a todo in your application it missed the id in your key-value pairs as you can see in the image. Kindly fix it :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @david63011 :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Following Linters
:heavy_check_mark: Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (david63011) [reply src/index.js:32]: @HAMMAS-SALEEM it works perfectly for me I will make a loom so you can see and maybe you can tell me what happened

Comment (david63011) [reply src/index.js:32]: @HAMMAS-SALEEM ok I will fix that tonight

Comment (HAMMAS-SALEEM) [reply src/index.js:32]: Hey @david63011
Sure dear you can tell me. That would be very nice of you :blush:

Comment (david63011) [reply src/index.js:32]: here the video that shows the edit function working normally

https://www.loom.com/share/961bf9cdcbcf406e92b3e49d84cbce86

Comment (david63011) [reply src/index.js:32]: @HAMMAS-SALEEM  it should be ok now

Comment (david63011) [reply src/index.js:33]: ok it is working now

**Review by ShadyShawkat — APPROVED** [review]
## Hi @david63011, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your code is clean and well documented ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**